src/pathfinding_service/api.py

```diff
@@ -362,10 +367,10 @@ def get(self) -> Tuple[dict, int]:
             "payment_address": to_checksum_address(self.pathfinding_service.address),
         }
         if info["message"] == DEFAULT_INFO_MESSAGE:
-            info["message"] = info["message"] + to_checksum_address(
+            info["message"] = info["message"] + to_checksum_address(  # type: ignore
```

Comment:
Why do we need the type ignore?

Reply:
Without that I get:
```
src/pathfinding_service/api.py:370: error: Unsupported left operand type for + ("object")
```

Reply:
I would prefer
```py
info["message"] = str(info["message"]) + ...
```
or
```py
assert isinstance(info['message'], str)
```
if mypy understands that. Type ignores are always a bit distracting.

Reply:
👍